src/company_gui.cpp

```diff
@@ -2237,6 +2237,10 @@ struct CompanyWindow : Window
 			}
 		}
 
+        if (TimerGameEconomy::UsingWallclockUnits()) {
+            this->GetWidget<NWidgetLeaf>(WID_C_DESC_INAUGURATION)->SetDataTip(STR_COMPANY_VIEW_INAUGURATED_TITLE_WALLCLOCK, STR_NULL);
+        }
+
```

Comment:
Good try, but we typically use a different way to change strings in GUI.

First, in the widget structure use a string that allows us to select strings as a parameter. In this case, you need to set two string parameters, so it should be STR_JUST_STRING2, like so:

```
NWidget(WWT_TEXT, COLOUR_GREY, WID_C_DESC_INAUGURATION), SetDataTip(STR_JUST_STRING2, STR_NULL), SetFill(1, 0),
```

Then choose the correct string in SetStringParameters:

```
case WID_C_DESC_INAUGURATION:
    if (TimerGameEconomy::UsingWallclockUnits()) {
		SetDParam(0, STR_COMPANY_VIEW_INAUGURATED_TITLE_WALLCLOCK);
		SetDParam(1, Company::Get((CompanyID) this->window_number)->inaugurated_year_calendar);
        SetDParam(2, Company::Get((CompanyID) this->window_number)->inaugurated_year);
	} else {
		SetDParam(0, STR_COMPANY_VIEW_INAUGURATED_TITLE);
		SetDParam(1, Company::Get((CompanyID)this->window_number)->inaugurated_year);
	}
    break;
```